UnitystationLauncher/ViewModels/MainWindowViewModel.cs

```diff
@@ -24,11 +33,18 @@ public string MaximizeToolTip
             set => this.RaiseAndSetIfChanged(ref _maximizeToolTip, value);
         }
 
-        public MainWindowViewModel(LauncherViewModel launcherVm)
+        public MainWindowViewModel(LoginViewModel loginVm, Lazy<LoginStatusViewModel> loginStatusVm, Lazy<LauncherViewModel> launcherVm,
+            AuthService authService)
         {
+            _loginStatusVm = loginStatusVm;
+            _loginVm = loginVm;
+            _authService = authService;
+            _launcherVm = launcherVm;
+            Content = _content = loginVm;
+            authService.AttemptingAutoLogin = false;
             _maximizeIcon = Geometry.Parse("M2048 2048v-2048h-2048v2048h2048zM1843 1843h-1638v-1638h1638v1638z");
             _maximizeToolTip = "Maximize";
-            Content = _content = launcherVm;
+            RxApp.MainThreadScheduler.ScheduleAsync((_, _) => CheckForExistingUserAsync());
```

Comment:
why are we disregarding the tuple?

Reply:
It's underused in the function so courtesy Told me to make it _